ubiquity/dell-bootstrap.py

```diff
@@ -1226,6 +1226,15 @@ def install(self, target, progress, *args, **kwargs):
                 if not found:
                     wfd.write("GRUB_DISABLE_OS_PROBER=true\n")
 
+            #set default recovery_type of 99_dell_recovery grub as 'hdd' for non-Wyse platforms
+            recovery_type = 'hdd'
+            #if wyse mode is on (dell-recovery/mode == 'wyse'), set the recovery_type to be 'factory'
+            #as Wyse platforms will always skip the "Restore OS Linux partition" dialog
+            if self.db.get('dell-recovery/mode')=='wyse':
```

Comment:
nit pick, can you put spaces between the == and other text?

Reply:
Sure, will do.